src/main/java/com/rtb/projectmanagementtool/project/ProjectController.java

```diff
@@ -26,60 +22,70 @@ public ProjectController(DatastoreService datastore) {
   }
 
   /**
-   * Gets all the projects in datastore that all specified users are in
+   * Gets all the projects in datastore that all users are in
    *
    * @param userIds the collection of userIds whose projects are to be retrieved
    * @return HashSet containing desired projects
    */
-  public HashSet<ProjectData> getProjects(Collection userIds) {
-
-    HashSet<ProjectData> projectContainer = new HashSet<ProjectData>();
+  public ArrayList<ProjectData> getAllProjects() {
+    return getProjects(0l, false);
```

Comment:
Good use of a wrapper method.  I would suggest creating a named constant for the `0l` though (maybe `ANY_USER_ID`).  Also please add comments to specify what the hard coded params are, eg:
```
return getProjects(/*userId=*/ ANY_USER_ID, /*getCreatorProjects=*/false);
```